test/sanity/validate-modules/ignore.txt

```diff
@@ -317,33 +317,9 @@ lib/ansible/modules/cloud/dimensiondata/dimensiondata_network.py E325
 lib/ansible/modules/cloud/dimensiondata/dimensiondata_network.py E326
 lib/ansible/modules/cloud/dimensiondata/dimensiondata_vlan.py E325
 lib/ansible/modules/cloud/dimensiondata/dimensiondata_vlan.py E326
-lib/ansible/modules/cloud/docker/docker_container.py E322
-lib/ansible/modules/cloud/docker/docker_container.py E324
-lib/ansible/modules/cloud/docker/docker_container.py E325
 lib/ansible/modules/cloud/docker/docker_container.py E326
```

Comment:
Do you want to fix the remaining docker_* E326 errors in this PR?

Reply:
Fix